mne/beamformer/_dics.py

```diff
@@ -14,16 +14,75 @@
 from ..minimum_norm.inverse import combine_xyz, _check_reference
 from ..source_estimate import _make_stc
 from ..time_frequency import CrossSpectralDensity, csd_epochs
-from ._lcmv import _prepare_beamformer_input, _setup_picks, _reg_pinv
+from ._lcmv import (_prepare_beamformer_input, _setup_picks, _reg_pinv,
+                    _subject_from_filter, _check_proj_match,
+                    _pick_channels_spatial_filter)
 from ..externals import six
 
 
 @verbose
-def _apply_dics(data, info, tmin, forward, noise_csd, data_csd, reg,
-                label=None, picks=None, pick_ori=None, real_filter=False,
-                verbose=None):
-    """Dynamic Imaging of Coherent Sources (DICS)."""
-    is_free_ori, _, proj, vertno, G =\
+def make_dics(info, forward, noise_csd, data_csd, reg=0.05, label=None,
+              pick_ori=None, real_filter=False, verbose=None):
+    """Compute Dynamic Imaging of Coherent Sources (DICS) spatial filter.
+
+    .. note:: Fixed orientation forward operators with ``real_filter=False``
+              will result in complex time courses, in which case absolute
+              values will be returned.
+
+    .. note:: This implementation has not been heavily tested so please
+              report any issues or suggestions.
```

Comment:
Can you check the restructured text syntax here? I think directives like note:: want a line break + indent. Let's make sure circle CI looks good.